coral-hive/src/main/java/com/linkedin/coral/hive/hive2rel/HiveSqlConformance.java

```diff
@@ -22,4 +22,9 @@ private HiveSqlConformance() {
   public boolean allowNiladicParentheses() {
     return true;
   }
+
+  @Override
+  public boolean isSortByAlias() {
```

Comment:
Looks like Hive also allows alias in `having` clause, but doesn't allow that in `group by` clause.
Could you also handle the `having` case (override `isHavingAlias`) in this PR?

Reply:
Good point, added in revision